charts/temporal/values/values.postgresql.yaml

```diff
@@ -40,6 +43,17 @@ server:
           #  enabled: true
           #  enableHostVerification: true
           #  serverName: _HOST_ # this is strictly required when using serverless CRDB offerings
+          #  caFile: /path/to/certs/<CA-file> # Here we assumed that caFile, certFile, keyFile are stored in one secret mounted as 'secret-with-certs' (see: server.additionalVolumes and server.additionalVolumeMounts sections).
+          #  certFile: /path/to/certs/<client-cert-file>
+          #  keyFile: /path/to/certs/<client-key-file>
+
+# additionalVolumes:
```

Comment:
These should be indented so that they are under `server`.

Reply:
@robholland Thanks, fixed.